scripts/eq_translations_check.py

```diff
@@ -10,8 +10,10 @@
 version = f"v{eq_translations.__version__}"
 
 try:
-    response = requests.get("https://api.github.com/repos/ONSdigital/eq-translations/releases").json()
-    latest_tag = response[0]['tag_name']
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-translations/releases"
+    ).json()
+    latest_tag = response[0]["tag_name"]
```

Comment:
I would check for a 200 response before checking access this.

Reply:
Sorted, thanks!👍